### PR TITLE
LPS-167893 HR_185 No asked to confirm the changes in the screen size

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPS-167893

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/components/SizeSelector.tsx [hunk 1]
```diff
@@ -218,18 +214,21 @@ function CustomSizeSelector({
...
 	previewRef,
 }: ICustomSizeSelectorProps) {
-	const {height, width} = useCustomSize();
-
-	const setHeight = useSetCustomHeight();
-	const setWidth = useSetCustomWidth();
+	const setCustomSize = useSetCustomSize();
 
 	const [alertMessage, setAlertMessage] = useState<string | null>(null);
+	const [height, setHeight] = useState<number>(
```
Comment (p2kmgcl) [inline modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/components/SizeSelector.tsx:220]: We can omit the type parameter (`<number>`) if we are already passing an initial value :smile: 

(same below)

File: modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/contexts/CustomSizeContext.tsx [hunk 2]
```diff
@@ -11,56 +11,40 @@ import React, {
...
+	const {customSize} = useContext(CustomSizeContext);
 
-	return {height, width};
+	return customSize;
 };
 
 function CustomSizeContextProvider({children}: {children: React.ReactNode}) {
-	const [height, setHeight] = useState<number>(
-		SIZES.custom.screenSize.height
+	const [customSize, setCustomSize] = useState<ScreenSize>(
```
Comment (p2kmgcl) [inline modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/contexts/CustomSizeContext.tsx:39]: Same as before :smile:

**Review by p2kmgcl — CHANGES_REQUESTED** [review]
Very nice. I just suggested a small change to remove a type declaration that can be inferred by the compiler.

**Review by p2kmgcl — APPROVED** [review]

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**